docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -229,16 +229,12 @@ To disable notifications about new Strapi releases, set the `config.notification
 
 #### Theme extension
 
-<!-- TODO: complete this section once design system is ready -->
-
 To extend the theme, use the `config.theme` key.
 
-::: Strapi Design System
-The default [Strapi theme](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/themes) defines various theme-related keys (shadows, colors…) that can be updated through the `config.theme` key in `./admin/src/app.js`.
+::: strapi Strapi Design System
```

Comment:
Is the extra `strapi` wanted?

Reply:
Yes it is. The first `strapi` makes sure we have a "Strapi" callout. This is something I should actually back port to thee `main` branch as we currently have a display bug in the live documentation.
![Screenshot 2022-02-15 at 15 28 55](https://user-images.githubusercontent.com/4233866/154082223-fdac0cf2-615c-49a0-b5f3-46889f9d8968.png)